superset-frontend/src/dashboard/components/gridComponents/Chart.test.jsx

```diff
@@ -122,17 +122,80 @@ test('should render a ChartContainer', () => {
   expect(getByTestId('chart-container')).toBeInTheDocument();
 });
 
-test('should render a description if it has one and isExpanded=true', () => {
-  const { container } = setup(
-    {},
-    {
-      dashboardState: {
-        ...defaultState.dashboardState,
-        expandedSlices: { [props.id]: true },
-      },
-    },
-  );
-  expect(container.querySelector('.slice_description')).toBeInTheDocument();
+describe('no description set', () => {
```

Comment:
I wanted to cover all 18 (three cases for expand slice, three for expand all and two for description existing - multiplied) test cases but did not want to create 18 tests cases. Therefore I went with code that creates tests. I will of course happily switch to individual test cases if required!